src/utils/image_utils.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import cv2
+
+
+def convert_to_grayscale(frame):
+    """
+    Function  to convert a frame to grayscale
+
+    :param frame: frame to be converted to grayscale
+    :return frame after converting to grayscale
+    """
+    return cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+
+
+def absolute_difference(curr_frame, prev_frame):
+    """
+    Function to find the absolute difference between two frames.
+    Takes the sum of pixel wise difference.
+    :param curr_frame: the current frame to be processed from the batch
+    :param prev_frame: the previous frame that was not skipped
+    """
+    curr_frame_grayscale = convert_to_grayscale(curr_frame)
+    prev_frame_grayscale = convert_to_grayscale(prev_frame)
+    frame_diff = cv2.absdiff(curr_frame_grayscale, prev_frame_grayscale)
+    return frame_diff.sum()
+
+
+def mse_difference(curr_frame, prev_frame):
+    """
+    Function to find the mean squared error difference between two frames.
+    :param curr_frame: the current frame to be processed from the batch
+    :param prev_frame: the previous frame that was not skipped
+    """
+    curr_frame_gray = convert_to_grayscale(curr_frame)
+    prev_frame_gray = convert_to_grayscale(prev_frame)
+    total_pixels = curr_frame_gray.shape[0] * curr_frame_gray.shape[1]
+    frame_diff = curr_frame_gray.astype('float') - prev_frame_gray
+    mse_diff = np.sum(frame_diff**2) / float(total_pixels)
+    return mse_diff
+
+
+def mask_background(frame, dim=21, threshold1=10, threshold2=200, 
```

Comment:
This method doesn't have any tests. 

If you feel testing the actual output is difficult, conisder using mocking and stubbing.